roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -38,9 +38,16 @@ void doBanner() {
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 /// Inspect cpu capabilities, and load architecture-specific libraries for RooFitCore/RooFit computations.
 void loadComputeLibrary() {
+  if (gEnv->GetValue("RooFit.LoadOptimisedComputationLibrary", 1) == 0) {
+    if (gDebug>0) std::cout << "RooFit.LoadOptimisedComputationLibrary is set to 0, so skipping loading any library" << std::endl;
```

Comment:
Does RooFit use `cout`? The rest of ROOT prefers `Info`, `Warning` etc, which can for instance be absorbed by the experiments' frameworks, or silenced by `gErrorIgnoreLevel`.

In other words: is there a logging framework that should be used here rather than `cout`, how is this done in other parts of RooFit?

Reply:
Yes, there is, but it's not initialised at this point.

Reply:
OK then please at least state then current function: `"In RoFit...::...::loadComputeLibrary(): ..."`